### ISSUE TITLE
failed to load il2cpp

### ISSUE THREAD

**Issue by Davidekiller1999** [ALEZ-DEV/Babylonia-terminal#28]
hey, thanks for your job!

when i launch the game it says "failed to load il2cpp"

**Comment by ALEZ-DEV** [comment]
Hey, thanks for your interest on the project!

You probably have faulty files, to check if your files are corrupted or not, you will need to check your game integrity, there's not such GUI feature for now, but there's a way to check and not redownload the whole game, please follow these steps :
1. go to `~/.babylonia-terminal` and open `babylonia-terminal-config` with any file editor
2. then you will pass the `is_game_installed` value and `is_game_patched` value to `false`
3. then restart the launcher and download one more time the game (it will just redownload the faulty files)
4. then you will be able to play the game!

Don't hesitate to tell me if anything goes wrong!
I will add this features when I get home

**Comment by Davidekiller1999** [comment]
Thanks for replying.
I tried but unfortunately it still gives me the same error

**Comment by ALEZ-DEV** [comment]
Did the launcher check the game files ?

**Comment by Davidekiller1999** [comment]
yes, it made me install 3.2 gb of missing files, but then nothing changed.
if it can be useful: I'm using manjaro and the penultimate nvidia drivers (the latest ones give problems with gnome)

**Comment by Davidekiller1999** [comment]
I tried this tonight and it worked. it took two tries but now the game seems to start! Thank you!

**Comment by ALEZ-DEV** [comment]
Good to hear that, I will keep this open until the feature is implemented

**Comment by MHYT-commits** [comment]
Hi.I am having the same issue.After Downloading some percentage of the faulty files it stop downloading.I tried multiple times and less than half of the faulty files are downloaded.
